package.json

```
@@ -19,7 +19,7 @@
     "setup": "yarn install && turbo run build",
     "start": "openmrs develop",
     "verify": "turbo run lint && turbo run test && turbo run typescript",
-    "prettier": "prettier --config prettier.config.js --write \"packages/**/*.{ts,tsx,css,scss}\" \"e2e/**/*.ts\"",
+    "prettier": "prettier --config prettier.config.js --write \"packages/**/*.{ts,tsx,css,scss}\" \"e2e/**/*.ts\" | grep -v unchanged",
```

This won't work on Windows.

```suggestion
    "prettier": "prettier --config prettier.config.js --write \"packages/**/*.{ts,tsx,css,scss}\" \"e2e/**/*.ts\" --list-different",
```

Accomplishes basically the same thing.